src/content/en/2021/cms.md

```diff
@@ -2,19 +2,614 @@
+  description="Column chart showing the median weight of fonts resources downloaded in each of the top 5 most adopted CMSs, for each device. The median WordPress site uses 121KB of fonts on mobile, and 159KB on desktop.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRzYkQYaBTpD5FKloDXsdb32Y2pjdxFZq_LoekECgTy53F-dRATm9wFFJ3dDVDS8_cGn0h2mKDBjgdM/pubchart?oid=750270305&format=interactive",
+  sheets_gid="740406389",
+  sql_file="resource_weights.sql"
+) }}
+
+To display text, web developers often choose to use a variety of fonts. Joomla delivers the fewest font bytes, with 75KB on mobile views, and Squarespace delivers the most with 212KB.
+
+## WordPress Specific
+
+WordPress is the most commonly used CMS today - almost 3 out of 4 sites built with a CMS are using WordPress, thus deserving further discussion.
+
+WordPress is an open source project, which has been around since 2003. Many sites built on WordPress use various themes and plugins, sometimes through page builders such as Elementor or Divi.
+
+The WordPress community maintains the CMS and services requirements for additional functionality through custom services and products (themes and plugins). This community has an outsized impact, with a relatively small number of people maintaining both the CMS itself and providing the additional functionality which makes WordPress sufficiently powerful and flexible that it can service most types of websites. This flexibility is important when explaining the market share, but also complicates the discussion around WordPress based site performance.
+
+Contributors from the WordPress community recently acknowledged the current state of performance, in this [proposal](https://make.wordpress.org/core/2021/10/12/proposal-for-a-performance-team/) to create a performance dedicated core team, which can hopefully improve the current performance of the average WordPress sites.
+
+### Adoption
+
+First, we’ll examine WordPress adoption by geography, across all sites in our dataset.
+
+{{ figure_markup(
+  image="wordpress-adoption-geo.png",
+  caption="WordPress adoption by geography.",
+  description="Bar chart depicting the adoption of WordPress in each of the 10 geographies with the most websites.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRzYkQYaBTpD5FKloDXsdb32Y2pjdxFZq_LoekECgTy53F-dRATm9wFFJ3dDVDS8_cGn0h2mKDBjgdM/pubchart?oid=625154641&format=interactive",
+  width=600,
+  height=579,
+  sheets_gid="43346414",
+  sql_file="core_web_vitals_by_geo.sql"
+) }}
+
+In the top 10 countries with the most sites in our dataset, WordPress had over 27% adoption. Spain had the highest WordPress adoption among these countries with 37% of mobile pages using WordPress, compared with Germany where _only_ 28% of mobile pages used WordPress.
+
+### Passing CWVs by Geo
+
+Next, let’s look at the amount of WordPress origins with passing Core Web Vitals, but this time, breakdown by geography, for mobile devices.
+
+{{ figure_markup(
```

Comment:
@rviscomi this image is cut in the middle (see WordPress origins passing CWV by geo here) - https://20211106t191715-dot-webalmanac.uk.r.appspot.com/en/2021/cms#wordpress-specific

could you fix it please? is it a height issue?

Reply:
Done